aws/resource_aws_kinesis_firehose_delivery_stream_test.go

```diff
@@ -1653,25 +1662,28 @@ resource "aws_kinesis_firehose_delivery_stream" "test" {
 
 var testAccKinesisFirehoseDeliveryStreamConfig_s3basic = testAccKinesisFirehoseDeliveryStreamBaseConfig + `
 resource "aws_kinesis_firehose_delivery_stream" "test" {
-  depends_on = ["aws_iam_role_policy.firehose"]
-  name = "terraform-kinesis-firehose-basictest-%d"
+  depends_on  = ["aws_iam_role_policy.firehose"]
```

Comment:
Can be done in later passes, but noting more cases here where quoted `depends_on` references are deprecated.

Reply:
Cleaned